runtime/ai/develop_model.go

```diff
@@ -54,8 +55,17 @@ func (t *DevelopModel) Handler(ctx context.Context, args *DevelopModelArgs) (*De
 		args.Path = "/" + args.Path
 	}
 
-	// Pre-invoke file listing to get context about existing models
 	session := GetSession(ctx)
+
+	if args.Create {
+		entries, err := t.Runtime.ListFiles(ctx, session.InstanceID(), args.Path+"*")
+		if err != nil {
+			return nil, err
+		}
+		args.Path = fileutil.GetPath(args.Path, entries)
+	}
```

Comment:
What is the use case for this? 

The `Path` passed to `DevelopModel` should ideally already be a correct `.yaml` path. If the `developer_agent` doesn't set that correctly, we should just return a useful error so it can self-correct. And if it does set it correctly, then I don't think we need the `Create` flag – it's already implied in the description for `Path`: 
```
Path string `json:"path" jsonschema:"The path of a .yaml file in which to create or update a Rill model definition."`
```
note "in which to create or update"